src/app/fdctl/run/tiles/fd_poh.c

```diff
@@ -519,13 +541,13 @@ fd_ext_poh_initialize( ulong         hashcnt_duration_ns, /* See clock comments
     FD_SPIN_PAUSE();
   }
   fd_poh_ctx_t * ctx = fd_ext_poh_write_lock();
-  
+
   ctx->hashcnt             = tick_height*hashcnt_per_tick;
   ctx->last_hashcnt        = ctx->hashcnt;
   ctx->reset_slot_hashcnt  = ctx->hashcnt;
-  ctx->reset_slot_start_ns = fd_log_wallclock();
+  ctx->reset_slot_start_ns = fd_log_wallclock(); /* safe to call from Rust */
 
-  fd_memcpy( ctx->hash, last_entry_hash, 32UL );
+  memcpy( ctx->hash, last_entry_hash, 32UL );
```

Comment:
I think memcpy is faster when the size is small and known at compile time because the compiler can do a better job optimizing it